net-p2p/deluge/deluge-2.1.1-r5.ebuild

```diff
@@ -57,6 +57,7 @@ RDEPEND="
 		>=dev-python/twisted-17.1.0[ssl(-),${PYTHON_USEDEP}]
 		>=dev-python/zope-interface-4.4.2[${PYTHON_USEDEP}]
 		dev-python/mako[${PYTHON_USEDEP}]
+		python_single_target_python3_13? ( dev-python/legacy-cgi[python_targets_python3_13] )
```

Comment:
$(python_gen_cond_dep '
	dev-python/legacy-cgi[${PYTHON_USEDEP}]
' 3.13)

Reply:
Also, needs a revbump as the dep is broken for 3.13.

Reply:
With the help of @parona-source we found an upstream patch which fixes the issue once and for all.

As per the edited commit, I have dropped py3_13 from the r5 and then created a r6 which includes the new patch.

Thanks for the advise on better python dep handling though as it will come in handy in future.

Reply:
If a deluge release with the patch is coming reasonable soon then using the legacy-cgi should be reasonable for the time being. 

But seeing that deluge hasn't had releases in two years...

Reply:
> If a deluge release with the patch is coming reasonably soon, then using the legacy-cgi should be reasonable for the time being.
> 
> But seeing that deluge hasn't had releases in two years...

-r6 includes the patch https://github.com/gentoo/gentoo/pull/39878/commits/bf7c9434f3955475422b2f0c8fe5d788dab0e1ee#diff-0acbe8576d5b8f195d90672191d25e2150f711ad9419fae2400a4137f0ec9faeR69 which has been working great here since testing.